src/pages/Search.tsx

```diff
@@ -17,13 +19,28 @@ const SearchBox = styled(Box)(({ theme }) => ({
   }
 }));
 
+const NoMapBox = styled(Box)(({ theme }) => ({
+  width: '100%',
+  height: '100%',
+  backgroundColor: theme.palette.common.white
```

Comment:
If we would have a dark mode, should it not be dark, opposed to white?

if so, you should get `.theme.palette.background.default`

Reply:
I was not aware of this feature. It has been updated. Thank you